easybuild/easyconfigs/g/GenomeTester4/GenomeTester4-0.4-intel-2018a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'GenomeTester4'
+version = '0.4'
+
+homepage = 'https://github.com/bioinfo-ut/GenomeTester4'
+description = "A toolkit for performing set operations - union, intersection and complement - on k-mer lists."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/bioinfo-ut/GenomeTester4/archive/']
+sources = ['Version_4_0.tar.gz']
+checksums = ['1849c4c6f2e6542679aa1831fa39d5be99786488309a3d2e674aa0b60dbe4863']
+
+prebuildopts = 'cd src && make clean && '
+
+files_to_copy = [
+    (['bin/glistcompare', 'bin/glistmaker', 'bin/glistquery',
+     'bin/gmer_caller', 'bin/gmer_counter'], "bin"),
+]
```

Comment:
@verdurin You can just make this:

```python
files_to_copy = ['bin']
```

To make it copy the whole `bin` directory, and then check for specific binaries in the sanity check (which you're already doing); that will avoid having to update this in case additional binaries are added in future versions...